docs/elements/hubspot/bulk.md

```diff
@@ -1,14 +1,27 @@
 ---
-heading: HubSpot Marketing
-seo: Bulk APIs | HubSpot Marketing | Cloud Elements API Docs
+heading: Hubspot Marketing
+seo: Bulk APIs | Hubspot Marketing | Cloud Elements API Docs
 title: Bulk APIs
 description: Add Bulk functionality to your app.
 layout: sidebarelementdoc
 breadcrumbs: /docs/elements.html
 elementId: 43
 parent: Back to Element Guides
-order: 35
+order: 30
 sitemap: false
 ---
 
-{% include bulk-apis.md %}
+# Bulk
+
+Cloud Elements Bulk support provides an option to upload a large number of resources, such as contacts, to a service provider all at once. You can use bulk to [download a csv or json](../../guides/bulk/download.html) file from a large number of records or [upload a csv or json](../../guides/bulk/upload.html) file to add multiple records. See the [Bulk Guide](../../guides/bulk/index.html) for more information on managing bulk.
+
+{% include note.html content="Cloud Elements leverages the native Hubspot bulk endpoints whenever available. When there is no bulk available from the provider, Cloud Elements provides a bulk service for uploading and downloading data from the endpoint. See <a href=#bulk-details>Bulk Details</a> for the type of bulk used." %}. When using the Native endpoints, performance is best when calls are limited to 100 or fewer. Additionally, when using this endpoint, please keep in mind that any errors with a single contact in your batch will prevent the entire batch from processing. If this happens, we'll return a 400 response with additional details as to the cause.
```

Comment:
Seems to be okay (https://cl.ly/0l002w2C180v). There was an awkward period there though so I've updated it. Thanks!